src/pages/wishlist_page/WishlistPage.jsx

```diff
@@ -0,0 +1,46 @@
+import "./wishlist_page.css";
+import { BasicCard, ButtonPrimary } from "../../components";
+
+const item = {
+  item_name: "La Porta Rossa Sulla Salita Art Print",
+  item_by: "Guido Borelli",
+  item_desc:
+    "Visit ten places on our planet that are undergoing the biggest changes today.",
+  item_price: 18,
+  item_original_price: 22,
+  item_rating: 4,
+};
+
+export const WishlistPage = () => {
+  return (
+    <main className="main">
+      <section className="page-header-section">
+        <div className="section-bg">
+          <img
+            src="https://d17h7hjnfv5s46.cloudfront.net/assets/build/images/banner_wishlist_no_connect.cbbca931.jpg"
+            alt=""
+          />
```

Comment:
Suggestion: Alt tag

Reply:
please look into it again it has alt tag @Ankur9669